### DISCUSSION TITLE
Line height setting

### DISCUSSION THREAD

**Discussion by bellebethcooper** [notaapp/talk#260] [Feature Requests]
Hello,
I'd love to see a setting for adjusting the line height in Nota. Thanks!

**Comment by astoilkov** [comment]
Hi Belle. Long time no see. Happy to see you logging an issue.

We are also thinking about this. We think we should also have a separate setting for line height in code blocks. Something like line height and code line height. We are still exploring this.

Thanks for writing!

**Reply by bellebethcooper** [reply]
Sounds good, thanks! I think I may have mentioned this to you before, so sorry for bugging you about it again. I used this setting a lot in Caret and miss it :)

Happy to see Nota progressing well lately!

**Reply by astoilkov** [reply]
No problem. It's a pleasure. We will probably first improve the design of Preferences (they are getting cluttered) first. After that, we will be able to more freely add some new preferences.

Thanks for the kind words! ❤️

**Reply by NoodleNodes** [reply]
Upvote for this please. I use bullets a lot (Roam Research holdover) and dense text becomes quite overwhelming. Would love to be able to adjust this by giving a little more breathing space between lines for readability.
